taichi/rhi/amdgpu/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# ./taichi/rhi/amdgpu/CMakeLists.txt
+
+set(AMDGPU_RHI amdgpu_rhi)
+add_library(${AMDGPU_RHI})
+target_sources(${AMDGPU_RHI}
+  PRIVATE
+    amdgpu_device.cpp
+    amdgpu_caching_allocator.cpp
+    amdgpu_context.cpp
+    amdgpu_driver.cpp
+  )
+
+target_include_directories(${AMDGPU_RHI}
+  PRIVATE
+    ${PROJECT_SOURCE_DIR}
+    ${PROJECT_SOURCE_DIR}/external/eigen
+    ${PROJECT_SOURCE_DIR}/external/spdlog/include
+    ${PROJECT_SOURCE_DIR}/external/glfw/include
```

Comment:
Wondering if you actually need `glfw` for AMDGPU, since we won't use AMDGPU to perform any rendering process?

Reply:
yeah, it is redundant